src/components/node/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { Meta as NewsStoryMeta } from '@/components/node/news_story'
+import { NodeTypes } from '@/types/node'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+
+/** General NodeProps to pass nodes into node components. */
+interface NodeProps {
+  node: NodeTypes
+  viewMode?: string
+}
+
+/** We expect each Node component to define and export information about itself to help with discoverability. */
+export interface NodeMetaInfo {
+  resource: string
+  component: ({ node, viewMode, ...props }: NodeProps) => JSX.Element
+  params: DrupalJsonApiParams
+}
+
+/** Collect all imported node meta information. */
+const nodeMetaIn: NodeMetaInfo[] = [NewsStoryMeta]
```

Comment:
Was it intentional to have NewsStoryMeta here?

Reply:
Yes, this will eventually collect all node 'meta' information. We would like to be able to look up component and such by resource, i.e. `node--news_story`, which we get from the JSON:API response, points to component `NodeNewsStory` for its rendering.